app/controllers/reviewables_controller.rb

```diff
@@ -196,6 +196,26 @@ def destroy
     render json: success_json
   end
 
+  def scrub
+    raise Discourse::InvalidAccess.new unless @guardian.is_admin?
+
+    params.require(:reason)
+
+    reviewable =
+      Reviewable.find_by(
+        id: params[:reviewable_id],
+        status: Reviewable.statuses[:rejected],
+        type: "ReviewableUser",
+      )
+    raise Discourse::NotFound.new if reviewable.blank?
+
+    raise Discourse::InvalidAccess.new if reviewable.payload["scrubbed_by"].present?
+
+    reviewable.scrub(current_user, params[:reason], @guardian)
```

Comment:
Because you @guardian is one of argument, instead of passing current_user, you can use `@guardian.current_user`